make.sh

```diff
@@ -0,0 +1,163 @@
+#!/bin/sh
+set -e
+
+# Script inspired by https://gist.github.com/szeidner/613fe4652fc86f083cefa21879d5522b
+
+PROGNAME=$(basename $0)
+WORKING_DIR=$(cd -P -- "$(dirname -- "$0")" && pwd -P)
+
+die() {
+    echo "$PROGNAME: $*" >&2
+    exit 1
+}
+
+usage() {
+    if [ "$*" != "" ] ; then
+        echo "Error: $*"
+    fi
+
+    cat << EOF
+Usage: $PROGNAME --bundle-id [BUNDLE_ID_PRODUCTION] --bundle-id-staging [BUNDLE_ID_STAGING] --app-name [PROJECT_NAME]
+Set up an iOS app from tuist template.
+Options:
+-h, --help                         display this usage message and exit
+-b, --bundle-id [BUNDLE_ID_PRODUCTION]  new production id (i.e. com.example.package)
+-s, --bundle-id-staging [BUNDLE_ID_STAGING]  new staging id (i.e. com.example.package.staging)
+-n, --app-name [PROJECT_NAME]          new app name (i.e. MyApp, "My App")
+EOF
+
+    exit 1
+}
+
+bundleidproduction=""
+bundleidstaging=""
+appname=""
```

Comment:
As we don't have a convention for shell script, but I think we can follow this [styleguide](https://google.github.io/styleguide/shellguide.html)
The variable names: Lower-case, with underscores to separate words
```suggestion
bundle_id_production=""
bundle_id_staging=""
project_name=""
```

Reply:
in the [styleguide](https://google.github.io/styleguide/shellguide.html#constants-and-environment-variable-names) document, it also mentions declaring constants and local variables using keywords (`readonly`, `local`), I think we can improve the script :D

Reply:
I'm using the same format from the android file https://github.com/nimblehq/android-templates/blob/develop/newproject.sh where lower var is a local variable and UPPER is constant. I agree with you that it's quite unreadable. Let's go with `local_var` format.